server/src/main/java/io/druid/metadata/SQLMetadataConnector.java

```diff
@@ -373,4 +373,35 @@ protected BasicDataSource getDatasource()
     return dataSource;
   }
 
+  private void createAuditTable(final IDBI dbi, final String tableName)
+  {
+    createTable(
+        dbi,
+        tableName,
+        ImmutableList.of(
+            String.format(
+                "CREATE TABLE %1$s (\n"
+                + "  id %2$s NOT NULL,\n"
+                + "  auditkey VARCHAR(255) NOT NULL,\n"
+                + "  author VARCHAR(255) NOT NULL,\n"
+                + "  emailid VARCHAR(255) NOT NULL,\n"
```

Comment:
there's a really inconsistent naming schema going on here, i think we shoudl use underscores everywhere